docs/user/rstudio/ide/guide/code/debugging.qmd

```diff
@@ -2,31 +2,31 @@
 title: "Debugging"
 ---
 
-Debugging in R is a broad topic. In this article, we focus specifically on the R debugging tools built into the RStudio; for more general advice on debugging in R (such as philosophy and problem-solving strategies), we recommend this resource from Hadley Wickham: [Debugging from Advanced R](https://adv-r.hadley.nz/debugging.html)
+Debugging in R is a broad topic. This documentation focuses specifically on the R debugging tools built into the RStudio. The [Advanced R - Debugging chapter](https://adv-r.hadley.nz/debugging.html) provides more general advice on debugging in R (such as philosophy and problem-solving strategies).
 
-Copied from <https://support.rstudio.com/hc/en-us/articles/205612627-Debugging-with-the-RStudio-IDE>
+In a very general sense, debugging is designed to help find bugs by figuring out where the code is not behaving as expected. 
 
-In a very general sense, debugging is designed to help you find bugs by figuring out where the code is not behaving in the way that you expect. To do this, you need to:
+To do this:
 
-1.  Begin running the code
+1.  Begin running the code.
 
-2.  Stop the code at the point where you suspect the problem is arising, and
+2.  Stop the code at the point where you suspect the problem is arising.
 
 3.  Look at and/or walk through the code, step-by-step at that point.
 
-We'll look at these last two tasks in detail.
+This documentation covers the last two tasks in detail.
 
 ## Entering debug mode (stopping)
```

Comment:
I thought this was because the Quarto version on Posit Connect is so old, but it's also occuring locally on 1.2. 

This issue was *not* resolved with the code-specific fix: https://rstudio.slack.com/archives/C01LQETLA7L/p1663858216818409

Reply:
Could the external link issue be related to this not being hosted on docs.rstudio.com yet?